clients/java/src/test/java/io/camunda/zeebe/client/util/RecordingGatewayService.java

```diff
@@ -435,6 +446,27 @@ public void onActivateJobsRequest(final Throwable error) {
     addRequestHandler(ActivateJobsRequest.class, () -> error);
   }
 
+  public void onStreamJobsRequest(final Throwable error) {
+    addRequestHandler(ActivateJobsRequest.class, () -> error);
```

Comment:
:x: Shouldn't this be `StreamActivatedJobsRequest`?

```suggestion
    addRequestHandler(StreamActivatedJobsRequest.class, () -> error);
```